R/formatter.r

```diff
@@ -232,32 +232,70 @@ scientific <- function(x, digits = 3, ...) {
 
 #' Ordinal formatter: add ordinal suffixes (-st, -nd, -rd, -th) to numbers.
 #'
-#' @return a function with single parameter x, a numeric vector, that
+#' @return A function with single parameter `x`, a numeric vector, that
 #'   returns a character vector
-#' @param x a numeric vector to format
+#' @param x A numeric vector of positive values to format.
+#' @param prefix,suffix Symbols to display before and after value.
+#' @param big.mark Character used between every 3 digits to separate thousands.
+#' @param rules Custom rules for computing ordinal indicators (list of regular
```

Comment:
Maybe: Named list of regular expressions, match in order. Name gives suffix, and value specifies which numbers to match.

Reply:
done